deployment/ci-cd-license-key.md

```diff
@@ -129,6 +130,25 @@ Also see [Using NuGet Keys](slug:deployment-nuget#using-nuget-keys) in the artic
         dotnet publish BlazorProjectName.csproj -c Release -o /app/publish /p:UseAppHost=false
     ````
 
+## Abort Deployment on License Key Error
+
+To avoid accidental [license watermarks and notifications on your live site](slug:installation-license-key#will-telerik-ui-for-blazor-work-with-an-expired-license-key), you can fail the application build and abort deployment when there is an issue with the license key. There are two ways to list the [Telerik license warning codes](slug:troubleshooting-license-key-errors#error-messages) to be treated as errors:
```

Comment:
```suggestion
To avoid accidental [license watermarks and notifications on your live site](slug:installation-license-key#will-telerik-ui-for-blazor-work-with-an-expired-license-key), you can fail the application build and abort deployment when there is an issue with the license key. There are two alternative ways to list the [Telerik license warning codes](slug:troubleshooting-license-key-errors#error-messages) to be treated as errors:
```